css/styles.css

```diff
@@ -0,0 +1,100 @@
+* {
+  margin: 0;
+  border: 0;
+  padding: 0;
+}
+
+h1 {
+  font-size: 2.5rem;
+}
+
+section {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.container-1 {
+  display: flex;
+  margin-left: 20px;
+  margin-bottom: 20px;
+  margin-top: 15px;
+}
+
+.container-2 {
+  font-family: Poppins, Arial, sans-serif;
+  margin-left: 20px;
+  margin-right: 20px;
+  display: flex;
+}
```

Comment:
- [ ] You made a lot of effort on the project 👏 . However, you can clearly see from my comparisons below on your design and that of the Figma template, that you have a couple of adjustments to do in fonts, background images, and margins.
- Aside from having a useful and great design, the UX of any website is beautiful due to the requirements being followed. Kindly check the images, and make appriorate adjustments 👍 

## Figma:
![image](https://user-images.githubusercontent.com/50990925/227406944-1cc9d20a-2085-44b9-8d71-91382b08b5b3.png)

## Your view:
![image](https://user-images.githubusercontent.com/50990925/227407365-53285d76-c78e-438a-88c7-691ba72efbd2.png)